### PR TITLE
feat(release): release official Ubuntu 22.04 (Jammy) packages

### PR DESCRIPTION
**Pull request by hutchic**
We've been testing Jammy for [~2 weeks](https://github.com/Kong/kong-build-tools/pull/567). Let's start officially releasing Jammy packages

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -81,7 +81,7 @@ pipeline {
                         sh 'echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin || true'
                         sh 'make setup-kong-build-tools'
                         sh 'make RESTY_IMAGE_BASE=debian KONG_TEST_CONTAINER_TAG="${GIT_BRANCH##*/}-debian" ADDITIONAL_TAG_LIST="${GIT_BRANCH##*/} ${GIT_COMMIT}" RESTY_IMAGE_TAG=11 release-docker-images'
-                        sh 'make RESTY_IMAGE_BASE=ubuntu KONG_TEST_CONTAINER_TAG="${GIT_BRANCH##*/}-ubuntu" RESTY_IMAGE_TAG=20.04 release-docker-images'
+                        sh 'make RESTY_IMAGE_BASE=ubuntu KONG_TEST_CONTAINER_TAG="${GIT_BRANCH##*/}-ubuntu" RESTY_IMAGE_TAG=22.04 release-docker-images'
```
Comment (gszr) [inline Jenkinsfile:84]: Is this removal intended? Could we leave both to avoid the breaking change? The 20.04 package is used, e.g., in the perf testing framework. @fffonion thoughts?

Comment (hutchic) [reply Jenkinsfile:84]: It is intended. Currently we only release per-commit docker images of the most recent LTS so Debian 11 / Alpine 3 therefore Ubuntu 22.04

**Review by gszr — APPROVED** [review]
